app/components/project-geometries/draw-lots-to-union.js

```diff
@@ -60,11 +60,19 @@ export default class DrawLotsToUnion extends Component {
 
       const drawStateCallback = () => {
         this.set('currentDrawing', draw.getAll());
+        this.set('drawMode', draw.getMode());
+
+        const selectedFeatures = draw.getSelected().features;
+        if (selectedFeatures.length === 1) {
+          this.set('selectedZoningFeature', selectedFeatures[0]);
+        } else {
+          this.set('selectedZoningFeature', undefined);
+        }
```

Comment:
Zoning is specific to underlying zoning and two other things - `draw-lots-to-union` is supposed to generic. This component could keep track of "selected" (clicked, but not necessarily drawn?)  features (as you have in line 65) but should have nothing to do with "domain" concerns like zoning, development site, etc.

Reply:
I've changed `selectedZoningFeature` to `selectedFeature`.